Controllers/Controller.php

```diff
@@ -54,12 +54,28 @@ public function __construct()
      *  Renders the given twig template with the given arguments and returns the result
      *
      * @param string $page Name of the view loaded
-     * @param array  $args Options for Twig's rendering
+     * @param array $args Options for Twig's rendering
      *
-     * @return Result generated by Twig
+     * @return string the result generated by Twig
+     * @throws \Twig_Error_Loader
+     * @throws \Twig_Error_Runtime
+     * @throws \Twig_Error_Syntax
      */
-    protected function render(string $page, $args=array()):string
+    protected function render(string $page, $args = array()):string
     {
         return $this->twig->render($page, $args);
     }
+
+    /**
+     * Controller method to redirect to login page if connection stops
```

Comment:
`Redirects to login page if the user is not connected, or returns the connected user`